tests/functional/export_v2/cram/minify-output.t

```diff
@@ -16,27 +16,29 @@ names to create larger file sizes with less recursion.
   > }
 
 A small tree is not automatically minified.
-The unminified output is 13KB which is considered acceptable.
+The unminified output is 15KB which is considered acceptable.
+
+  $ echo "$(generate_newick 10);" > small_tree_raw.nwk
+
+Use augur refine to add internal node names
+  $ ${AUGUR} refine --tree small_tree_raw.nwk --output-tree small_tree.nwk  &>/dev/null
```

Comment:
Suggestion: do this in `generate_newick` directly. I just tested locally and something like this seems to work:

```diff
- echo "($(generate_newick $((n-1))),$prefix$n)"
+ echo "($prefix$n,$(generate_newick $((n-1)))):INTERNAL_$n"
```

This runs much faster than `augur refine`, especially for the large tree.

Reply:
Without running `refine` we fail validation due to the absence of any divergence values on the tree.

> I just tested locally and something like this seems to work:

Do you have a patch which passes the tests?

Reply:
I would skip validation since validity should be tested elsewhere. Patch: d1c3997af6c0c659aaf38a0e47d1486e6fd58f95